### ISSUE TITLE
Improve bash_alias update process

### ISSUE THREAD

**Issue by ghost** [dogsbodytech/workstation-tools#5]
Currently you need to manually remove commands from bash_aliases for it to be re-created. The intention was to stop duplicate commands being set up.

We should be able to fix this update process issue and retain the no-duplicates aim by adding the following command before we set up new aliases:
`sed -i '/randpw() { for i in/d' ~/.bash_aliases`

**Comment by dogsbody** [comment]
Would prefer if we used `sed -i '/^randpw() /d' ~/.bash_aliases` as it doesn't assume the way that the old alias is setup.

**Comment by ghost** [comment]
Yep I suggest we base the `sed` line off of the `grep` commands.

For whoever picks this up, these are the jobs that need doing off of this:
- alias removal commands set up for all aliases. Including the following commands:
  - markdown
  - to_uuid
  - randpw
  - from_epoch
  - html_character_parser
  - slackpretty
  - musicpi
- Remove `grep -q -F ` as it is now redundant
